pinc/charsuite-symbols-collection.inc

```diff
@@ -0,0 +1,53 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+$charsuite = new CharSuite("symbols-collection", _("Symbols collection"));
+$charsuite->description = _("Astronomical, zodiac and a limited number of apothecary and music symbols. The zodiac symbols are provided with the variation selector, VS15, to try to force them to display as ordinary charcters instead of as emoji. This may not work for all fonts.");
```

Comment:
what's our opinion on serial commas? :D

Reply:
That they're used inconsistently across the site. What would you suggest here? \<grin\>

Reply:
I have no suggestion. I just usually use it so it stuck out to me. I approved because this shouldn't hold anything up. Just pointing it out in case there was some standard or you had a preference.

Reply:
After reviewing the wikipedia page on serial commas and re-reading the blurb, I added it here (but neglected to add it in the `.md` file, which I've chosen to leave as-is unless there are mutterings on inconsistency :D).